waku/waku_peer_exchange/protocol.nim

```diff
@@ -57,7 +58,7 @@ type
     requestRateLimiter*: RequestRateLimiter
 
 proc request*(
-    wpx: WakuPeerExchange, numPeers: uint64, conn: Connection
+    wpx: WakuPeerExchange, numPeers = DefaultPXNumPeersReq, conn: Connection
```

Comment:
Is it necessary to apply default here? ... and for other variants?

Reply:
Over here it's not necessary! But added it because given that we now defined a default, it would make sense for that number to be used whenever no value is provided